File: php/Depeer.php
<?php

class Depeer {

    // private attributes 
    private $id;
    private $name;
    private $bidang;
    private $partai;
    private $foto;

    public function __construct($id = "", $name = "", $bidang = "", $partai = "", $foto = "") {
        $this->id = $id;
        $this->name = $name; 
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->foto = "foto/" . $foto;
    }

    // Getter and Setter 
    public function getId() {
        return $this->id;
    }

    public function setId($id) {
        $this->id = $id;
    }

    public function getName() {
        return $this->name;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function getBidang() {
        return $this->bidang;
    }

    public function setBidang($bidang) {
        $this->bidang = $bidang;
    }

    public function getPartai() {
        return $this->partai;
    }

    public function setPartai($partai) { 
        $this->partai = $partai;
    }

    public function getFoto() {
        return "<img src='" . $this->foto . "' width='100' height='100'>";
    }

    public function setFoto($foto) {
        $this->foto = $foto;
    }

}

?>

File: python/Depeer.py
# class declaration
class Depeer:

    # private attribute
    __id = ""
    __name = ""
    __bidang = ""
    __partai = ""

    def __init__(self, id = "", name = "", bidang = "", partai = ""):
        self.__id = id
        self.__name = name
        self.__bidang = bidang
        self.__partai = partai

    # Getter and Setter
    def getId(self):
        return self.__id
    def setId(self, id):
        self.__id = id

    def getName(self):
        return self.__name
    def setName(self, name):
        self.__name = name

    def getBidang(self):
        return self.__bidang
    def setBidang(self, bidang):
        self.__bidang = bidang

    def getPartai(self):
        return self.__partai
    def setPartai(self, partai):
        self.__partai = partai